command.go

```diff
@@ -33,6 +34,8 @@ type Command struct {
 	procIDs []string
 	remote  remoteCommandOptions
 	makep   ProcessConstructor
+
+	proc Process
```

Comment:
I think this doesn't work in the RunParallel case. would it be possible to pass the process around?